core/domain/feedback_validators.py

```diff
@@ -152,6 +152,22 @@ def _get_external_id_relationships(cls, item):
             )
         return field_name_to_external_model_references
 
+    @classmethod
+    def _validate_text(cls, item):
+        """Validate that text length is less than 10000 characters.
+
+        Args:
+            item: datastore_services.Model. GeneralFeedbackMessageModel to
+                validate.
+        """
+
+        if item.text:
+            if len(item.text) > 10000:
```

Comment:
Can't we import the MAX_REVIEW_MESSAGE_LENGTH constant for use here? And refer to it in the docstring as well?

Reply:
Done.